views/main/index.ejs

```diff
@@ -0,0 +1,10 @@
+<h1>Link Shortener</h1>
+
+<form action="/links" method="POST">
+  <div class="form-group">
+  <input type="text" class="form-control" id="url" name="url" placeholder="enter URL here" required>
```

Comment:
res.redirect needs http:// or it wont redirect to the address. By setting the input type to url instead of text it does a front end validation and alerts the user that the address is invalid. type="url"